app/plugins/adp-backend/seedData/delivery_programme.js

```diff
@@ -4,6 +4,14 @@ const TABLE_NAME = 'delivery_programme';
  * @param {import('knex').Knex} knex
  */
 exports.seed = async function seed(knex) {
+
+  const recordsWithEmptyFields = await knex(TABLE_NAME)
+  .select('name')
+  .where(function() {
+    this.where('description', '')
+    .orWhere('delivery_programme_code', '')
+});
```

Comment:
The indentation here is a bit off, I think these lines need moving in by 1 indent